modules/tinymce/src/core/main/ts/api/html/Schema.ts

```diff
@@ -762,13 +746,11 @@ const Schema = (settings?: SchemaSettings): Schema => {
       children[name] = element.children;
     });
 
-    // Switch these on HTML4
-    if (settings.schema !== 'html5') {
```

Comment:
This we weren't too confident on what should happen, but as this is what was happening previously it's the closest to what most people would be expecting. In addition to that, if you go by the comment then this check wouldn't have worked for `html5-strict` and in addition to that in HTML 4 `b` and `strong` are very different (see below), so it likely wasn't ever correct anyways 🤷 . So for now we're leaving it as is and we'll discuss later when more people are back as to what exactly this should do.

> In HTML 4, the `strong` tag is used for indicating stronger emphasis than the `em` tag. The strong tag surrounds the emphasised word/phrase.